WordPress/gutenberg
# Video: add spacing block supports

Related:
- https://github.com/WordPress/gutenberg/issues/43241
- https://github.com/WordPress/gutenberg/issues/43243

## What?

Enabling spacing support for the Video block

## Why?

To create consistency across blocks.

## How?

Adding the relevant block supports in block.json

## Testing Instructions


1. Load the block editor and add a Video block. Add some content or use the example block code below.
2. Confirm that the dimensions control panel is there
3. Add margin and padding.
4. Save and confirm the styles appear on the frontend.
5. Test the new support works for the block via theme.json and global styles. See example JSON below.


```json
	"styles": {
		"blocks": {
			"core/video": {
				"spacing": {
					"margin": {
						"top": "30px",
						"bottom": "40px"
					}
				}
			}
		}
	}
```


https://user-images.githubusercontent.com/6458278/185363765-a2a8bdf5-2318-4b36-89a5-585359a739cf.mp4
